src/frontend/src/flows/authorize/index.ts

```diff
@@ -32,27 +32,26 @@ export const authnTemplateAuthorize = ({
         })
       : undefined;
 
-  const wrap = (slot: string) => html`
-    <div class="l-stack t-centered">
+  const wrap = (title: string) => html`
+    <div class="t-centered">
+      <h1 class="t-title t-title--main">${title}</h1>
       <p class="t-lead">
-        ${slot}<br />
-        <span class="t-strong">${origin}</span><br />
+        to continue to <a href="${origin}" target="_blank">${origin}</a><br />
```

Comment:
Is `noopener noreferrer` still relevant nowadays? CC @peterpeterparker

Reply:
I think it is, good point.